Source/JavaScriptCore/wasm/WasmOperations.cpp

```diff
@@ -799,13 +1010,14 @@ JSC_DEFINE_NOEXCEPT_JIT_OPERATION(operationAllocateResultsArray, JSArray*, (JSWe
     VM& vm = instance->vm();
```

Comment:
Let's change `operationAllocateResultsArray` to use JSC_DEFINE_JIT_OPERATION, and add exception handling to the code calling this function.